MANIFEST.in

```diff
@@ -1,4 +1,5 @@
 include LICENSE
+include *.toml
```

Comment:
I don't think we want to include this file in source distributions on pypi, so please remove this line.